doc/source/ray-contribute/docs.ipynb

```diff
@@ -32,7 +32,9 @@
     "If you want to contribute to the Ray documentation, you'll need a way to build it.\n",
     "You don't have to build Ray itself, which is a bit more involved.\n",
     "\n",
-    "You just have to [clone the Ray repository](https://docs.ray.io/en/master/ray-contribute/development.html#clone-the-repository), then [prepare the Python environment](https://docs.ray.io/en/master/ray-contribute/development.html#prepare-the-python-environment).\n",
+    "Follow the instructions in [Building Ray from Source](https://docs.ray.io/en/master/ray-contribute/development.html#building-ray-from-source) up to and including the following:\n",
+    "1. [Fork the Ray repository](https://docs.ray.io/en/master/ray-contribute/development.html#fork-the-ray-repository)\n",
+    "2. [Prepare a Python Virtual Environment](https://docs.ray.io/en/master/ray-contribute/development.html#prepare-a-python-virtual-environment)\n",
```

Comment:
If we're going to be following the same steps as the section above above, I think this should be step 3, with step 2 being "Clone the repo."

Reply:
To clarify, you're proposing the flow to be:
1. Fork the repo
2. Clone the repo
3. Prepare a Python virtual environment

I have these two steps (fork, prepare) to correspond to the two sections in the "Building Ray from Source" doc.